Profiler/TriggerProfiler.md

```diff
@@ -0,0 +1,61 @@
+## Diagnosing a WebAPI experiencing intermittent high CPU using the Application Insights Profiler
+The Application Insights Profiler is a great tool to investigate code level performance problems. It collects traces every hour for two minutes allowing you to do proactive performance analysis of your applications. While proactive performance analysis is a very useful activity to reduce performance bottle necks in a steady state, it can be more challenging to use when the problem is reactive and intermittent in nature. In this tutorial, we will take a look at a new capability recently added - trigger based profiling that is very handy in those reactive scenarios. 
+
+### Scenario Background
+In this tutorial we will diagnose an application (DiagService) running in Azure App Services that is experiencing slowdowns due to intermittent high CPU conditions. We can see what the CPU activity for the application looks like by using the Application Insights metrics explorer and choosing the CPU metric.
+
+![](https://user-images.githubusercontent.com/15442480/66504200-82eea400-ea7d-11e9-9ca9-01ba0d1749cf.png)
```

Comment:
Add a description for the image please, for accessibility. Same for the rest of the images.